docs/tutorials/hello-world.rst

```diff
@@ -30,6 +30,11 @@ save it as ``rockcraft.yaml``:
 .. literalinclude:: code/hello-world/rockcraft.yaml
     :language: yaml
 
+This file permit to build a ROCK with only the ``hello`` package slice inside,
+using :ref:`chisel_explanation`. For more information about the ``parts``
+section, check :ref:`ref_parts`. The other fields of the file are metadata
+which describes the ROCK; for more information about the available check
```

Comment:
this sentence doesn't read well. seems to be missing something..."available..." feature? fields?